tools/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN cd /tmp \
   && mkdir -p /tmp/build/.modcache \
   && cd /tmp/tools \
   && go install -trimpath -tags=tools github.com/golangci/golangci-lint/cmd/golangci-lint \
-  && chmod -R 777 /tmp/build/
+  && chmod -R 777 /tmp/build/ \ 
+  && git config --global --add safe.directory /build
```

Comment:
Doesn't seem to be working just yet...

Reply:
Fixed in #427

I am still not sure how it can be solved in lnd without making the folder "safe" for git :confused: